### ISSUE TITLE
Installing new updates doesn't overwrite old ones and occupies double the space.

### ISSUE THREAD

**Issue by fmbriggs** [blawar/tinleaf#21]
When installing updates, older updates are not uninstalled and still consume space on the device. Goldleaf does not have this issue as it detects and overwrites old updates.

Ideally there should be a tool to scan and detect obsolete updates. Currently, you have to manually open each title in Goldleaf or DBI and check them.

**Comment by blawar** [comment]
Why not use Lithium, Tinfoil, or DBI that does not have this issue?

**Comment by fmbriggs** [comment]
> Why not use Lithium, Tinfoil, or DBI that does not have this issue?

I need the PC-side UI of NS-USBLoader where I can select only a few files to be transferred to the Switch. It's a pain to scroll through thousands of files on the Switch in the other apps.

**Comment by blawar** [comment]
Boot tinfoil or lithium, connect usb cable, a MTP device shows up on your PC, copy the NSP/NSZ you want to install to the install folder in the MTP device and they install while picking from the PC side.

**Comment by fmbriggs** [comment]
I'll try that, thanks. Still, it's a bug to be fixed. Is there a way to clean up duplicate updates that are already installed?

**Comment by blawar** [comment]
You're right, it is a bug.  And I will fix it eventually, its just a low priority.

Rebooting your switch will reclaim the space.

**Comment by fmbriggs** [comment]
I used Lithium 11.00 and pasted a file in Install folder over MTP. Nothing happens after it's done. I can't see any existing files in the Install folder either. The other folders like Firmware and System have existing files.

**Comment by iMonZ** [comment]
> Why not use Lithium, Tinfoil, or DBI that does not have this issue?

Lithium and Tinfoil doesnt work with hekate!
